Mathlib/RingTheory/Valuation/Extension.lean

```diff
@@ -160,6 +158,61 @@ theorem instIsLocalHomValuationInteger {S ΓS : Type*} [CommRing S]
 
 end integer
 
+section AlgebraInstances
+
+open IsLocalRing Valuation ValuationSubring
+
+variable {K L Γ₀ Γ₁ : outParam Type*} [Field K] [Field L] [Algebra K L]
+  [LinearOrderedCommGroupWithZero Γ₀] [LinearOrderedCommGroupWithZero Γ₁] (vK : Valuation K Γ₀)
+   (vL : Valuation L Γ₁) [vK.HasExtension vL]
+
+local notation "K₀" => vK.valuationSubring
+local notation "L₀" => Valuation.valuationSubring vL
```

Comment:
```suggestion
local notation "K₀" => Valuation.valuationSubring vK
local notation "L₀" => Valuation.valuationSubring vL
```
to be consistent. This is better than having both `vK.valuationSubring` and `vL.valuationSubring` because this latter, dot-notation style, doesn't actually get used in pretty-printing -- I checked at the `rfl` term inside `coe_algebraMap_valuationSubring_eq`.

Any reason to prefer `valuationSubring` over `Valuation.integer`? Maybe this should even be phrased using `Valuation.Integers` so that it is more general than the "inherent" subring